doc/data/messages/c/cyclic-import/bad/bad.py

```diff
@@ -0,0 +1,9 @@
+from .bad2 import count_to_two  # [cyclic-import]
```

Comment:
~~Can we take into account the fact that if we want to do multiple examples properly displayed we're going to want to have bad.py, bad2.py, bad3.py (...) too ? Maybe the display can change if there's no good2.py, (...)  matching the bad2.py (...) ?~~ Edit: Did not notice it's in a directory, I think it's compatible.